client/src/pages/tasks.css

```diff
@@ -0,0 +1,33 @@
+.tasks {
+  height: 100%;
+  display: flex;
+  align-items: flex-start;
+  padding: calc(var(--q-space-base) * 4) calc(var(--q-space-base) * 6);
+}
+
+#tasks--input {
+  width: 20%;
+}
+
+#tasks--output {
+  width: 65%;
+  margin-left: 5%;
+}
+
+.tasks--output--json {
+  width: 800px;
```

Comment:
this should not be a fixed width if it's container is using a percentage width. why not just use 100% width (e.g. the default for a block element)?